package.json

```diff
@@ -64,10 +67,13 @@
   },
   "scripts": {
     "build": "tsc",
+    "initdb": "ts-node ./src/tools/initdb",
```

Comment:
The current build failure is caused by this not working on the `prod` image, since it's no longer TypeScript.